test/metabase/events/audit_log_test.clj

```diff
@@ -10,24 +10,19 @@
    [metabase.lib.schema.id :as lib.schema.id]
    [metabase.models
     :refer [Card Dashboard DashboardCard Metric Pulse Segment]]
+   [metabase.public-settings.premium-features-test :as premium-features-test]
    [metabase.test :as mt]
    [metabase.util :as u]
-   [toucan2.core :as t2]
    [toucan2.tools.with-temp :as t2.with-temp]))
 
 (set! *warn-on-reflection* true)
 
 (comment events.audit-log/keep-me)
 
-(defn latest-event
-  ([topic]
-   (latest-event topic nil))
-
-  ([topic model-id]
-   (t2/select-one [:model/AuditLog :topic :user_id :model :model_id :details]
-                  :topic    topic
-                  :model_id model-id
-                  {:order-by [[:id :desc]]})))
+(clojure.test/use-fixtures :once
+  (fn [test-fn]
+    (premium-features-test/with-premium-features #{:audit-app}
+      (test-fn))))
```

Comment:
👍  

We should probably use `use-fixtures` more than we currently are. I usually just see it used to initialize the test DB.